tc/tc_util.h

```diff
@@ -66,9 +66,12 @@ const char *get_tc_lib(void);
 struct qdisc_util *get_qdisc_kind(const char *str);
 struct filter_util *get_filter_kind(const char *str);
 
+int read_prop(char *dev, char *prop, long *value);
```

Comment:
I looked at where we could move the function and none of the header files included in these two files seem to have much in common. So the best we can do is move to a file which has to be included both in `iptuntap.c` and `tc_util.h`. Would that be ok?